README.md

```diff
@@ -1,18 +1,23 @@
 # RDA Game
 ![lint](https://github.com/RSA-Bots/RDA-Game/workflows/lint/badge.svg)
 - Live Place: [link](#)
-- Development Place: [link](#)
+- Development Place: [link](https://www.roblox.com/games/9304618061/RDA-Game-Dev)
 
 ## Getting Started
 1. Install Git
 	- Run `winget install Git.Git` if **winget** is available.
 	- Go to [https://git-scm.com/download](https://git-scm.com/download) otherwise.
-2. Install Tooling
+2. Next Clone the Repo
+	- Open a terminal and navigate to a directory where you want to store this repository.
+    - Run `git clone https://github.com/RSA-Bots/RDA-Game`
+    - This will then clone the repo to your filesystem.
+    - Once the cloning is finished, in your terminal navigate into `./RDA-Game`.
```

Comment:
Maybe "git checkout develop" should be mentioned here, although the primary branch is dev so not sure if that's needed

Reply:
I think it will go in the contributing file, where full details for how to properly pull, ensure their local repo is up to date, and create their own branches will be.

Reply:
Especially as this is meant for getting the tooling setup.